solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,174 @@
+import os
+import sys
+import json
+
+ATM_FILE_PLACE = 1
+COSTUMER_CHOICE_OPTIONS = "1 - View balance \n2 - Withdraw \n3 - Deposit " \
+                          "\n4 - Change password \n5 - Exit\n"
+
+costumer_id = ""
+costumer_password = ""
+
+
+def check_convert_to_float(amount):
+    """Check if variable can be converted to float.
+
+    Args:
+        amount (string): Costumer's input.
+
+    Returns:
+        float: Change the type of the variable 'amount' and return it,
+            else if couldn't- return None.
+    """
+    try:
+        convert_type = float(amount)
+        return convert_type
+    except ValueError:
+        return None
+
+
+def cash_withdraw(costumers_data):
+    """Withdraw amount of money from the ATM .
+
+    Args:
+        costumers_data (dictionary): Bank costumer data.
+
+    Returns:
+        boolean: True for success withdraw action, False otherwise.
+    """
+    amount_to_withdrawal = input("Amount to withdrawal: ")
+    check_amount = check_convert_to_float(amount_to_withdrawal)
+    if check_amount is not None:
+        balance_now = float(costumers_data[costumer_id]['balance'])
+        if 0 < check_amount <= balance_now:
+            new_balance = balance_now - check_amount
+            costumers_data[costumer_id]['balance'] = new_balance
+            return True
+
+    print("Failed - check amount\n")
+    return False
+
+
+def cash_deposit(costumers_data):
+    """Deposit money to the bank.
+
+    Args:
+        costumers_data (dictionary): Bank costumer data.
+
+    Returns:
+        boolean: True for success deposit action, False otherwise.
+    """
+    amount_to_deposit = input("Amount to deposit: ")
+    check_amount = check_convert_to_float(amount_to_deposit)
+    if check_amount is not None:
+        if check_amount > 0:
+            balance_now = float(costumers_data[costumer_id]['balance'])
+            new_balance = balance_now + check_amount
+            costumers_data[costumer_id]['balance'] = new_balance
```

Comment:
set `'balance'` and `'password'` into CONSTANT variables. This will make your life easier in future changes because you are only will have to change in one place.